decidim-core/app/uploaders/decidim/avatar_uploader.rb

```diff
@@ -8,11 +8,11 @@ class AvatarUploader < ImageUploader
     process :validate_dimensions
 
     version :big do
-      process resize_and_pad: [500, 500]
+      process resize_to_fit: [40, 40]
     end
 
     version :thumb do
-      process resize_and_pad: [100, 100]
+      process resize_to_fit: [20, 20]
```

Comment:
Same here with image distorsion, I wonder why we didn't see it before. I can dig in further to confirm.

Reply:
Same resolution doubling here